packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -121,8 +167,78 @@ class ExpansionPanelList extends StatelessWidget {
   /// The duration of the expansion animation.
   final Duration animationDuration;
 
+  //Whether multiple panels can be open simultaneously
+  final bool _allowMultiplePanelsOpen;
+
+  @override
+  State<StatefulWidget> createState() => new _ExpansionPanelListState();
+}
+
+class _ExpansionPanelListState extends State<ExpansionPanelList> {
```

Comment:
Please take another pass on formatting these files - see the style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo